src/interfaces/task.ts

```diff
@@ -0,0 +1,16 @@
+export default interface test {
```

Comment:
use descriptive name please

Reply:
I really like it since you choose it yourself